File: test/ayush.dev/python/pipelines/PythonLivy/code/pythonlivy/graph/Reformat_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from pythonlivy.config.ConfigStore import *
from pythonlivy.udfs.UDFs import *

def Reformat_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0

File: test/ayush.dev/scala/pipelines/CreatedFromUsedAccount/code/src/main/scala/io/prophecy/pipelines/createdfromusedaccount/graph/Reformat_1.scala
package io.prophecy.pipelines.createdfromusedaccount.graph

import io.prophecy.libs._
import io.prophecy.pipelines.createdfromusedaccount.config.ConfigStore._
import io.prophecy.pipelines.createdfromusedaccount.udfs.UDFs._
import io.prophecy.pipelines.createdfromusedaccount.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_1 {
  def apply(spark: SparkSession, in: DataFrame): DataFrame = in
}
